packages/augur-ui/src/modules/app/actions/init-augur.ts

```diff
@@ -263,6 +263,77 @@ export function connectAugur(
   };
 }
 
+function showIndexedDbSize() {
```

Comment:
put this in a different file. init-augur is busy, imo
create a file and put in util dir.